rhel.Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
+FROM rhel8/go-toolset:1.11.13-18 as builder
```

Comment:
this image is not public it can't be used in Eclipse Che

Reply:
@nickboldt I have no idea why this image was added in general without proper ci setup? Could you clarify?

Reply:
it's not used in Che. It's a secondary image that makes it easier for upstream to remember that there's a downstream, so if they update Dockerfile they MIGHT remember to apply similar change to rhel.Dockerfile. But it's not strictly an artifact that's USED in the Che build process

Reply:
yeah, but for devfile & plugin registry we do build rhel image as part of CI. Does it make sense to do the same for `che-machine-exec`. In the current state presense of this  RHEL image is misleading IMO

Reply:
well then there are two options:

a) keep it up here i che-machine-exec and DO build it as part of CI

b) remove it and hope that not having a rhel version in upstream causes any downstream impacts when I have to absorb changes from up to down and can't find an SME online (due to TZ conflicts).

I'd prefer (a). In fact I'd prefer if building on UBI8 was the default, not an option, but hey. :)